test/fixtures/basic_model.conf

```diff
@@ -4,6 +4,9 @@ r = sub, obj, act
 [policy_definition]
 p = sub, obj, act
 
+[role_definition]
```

Comment:
if `g` is added, then it's not the "basic model". See built-in examples here: https://github.com/casbin/casbin/tree/master/examples

If a new model is created, make sure a suitable name is given.

Reply:
Should I just change it to rbac since basic_model will fail when using `SavePolicy()`. I refered to this issue in Gitter earlier on.

Reply:
`g` is an optional section in Casbin model. Any adapter should work with the basic model (with no `g`)

Reply:
If this is a bug, we should fix the bug in the adapter, instead of adding `g` just to avoid the bug (and without using roles).

Reply:
Basically, if `g` is not provided it will raise `TypeError: groupingPolicyAST is not iterable`. This should happen in every implementation JS/TS implementation of the adapter.

Reply:
We should allow `groupingPolicyAST ` to be empty.

Reply:
We need to fix all adapters then.

Reply:
Okey, I think I can solve this by just checking whether `groupingPolicyAST` is a Map and if it's not just apply an empty Map into it.

Should I use rbac or basic_model for the tests or both?

Reply:
Both, use the official example.